modules/rules_nodejs/4.5.1/source.json

```diff
@@ -1,4 +1,5 @@
 {
-    "integrity": "sha256-IazjGVbtLULE0aceyb9E6I7cjU5tPoj3dDpW7h/JW7E=",
-    "url": "https://github.com/bazelbuild/rules_nodejs/releases/download/4.5.1/rules_nodejs-core-4.5.1.tar.gz"
+    "integrity": "sha256-rjfg/8hmmyV4k8jTUlVNFH0F+e+xxYDTujhfXfMUPyU=",
+    "strip_prefix": "rules_nodejs-4.5.1",
+    "url": "https://github.com/bazelbuild/rules_nodejs/archive/refs/tags/4.5.1.tar.gz"
```

Comment:
this is wrong, the original was a release artifact and shouldn't be changed. it's much smaller than the entire source repo